src/wam2layers/analysis/visualization.py

```diff
@@ -67,13 +67,13 @@ def _plot_precip(config, ax):
 
 def _plot_evap(config, ax):
     """Return subplot with tracked evaporation."""
-    region = load_region(config)
+    region = load_tagging_region(config)
```

Comment:
```suggestion
    region = load_tracking_region(config)
```

Right?

Reply:
No, I think this used to plot only the domain for which is tagged. But it might not make sense with all the new options.